src/util/Injected.js

```diff
@@ -5,15 +5,15 @@ exports.ExposeStore = (moduleRaidStr) => {
     eval('var moduleRaid = ' + moduleRaidStr);
     // eslint-disable-next-line no-undef
     window.mR = moduleRaid();
-    window.Store = Object.assign({}, window.mR.findModule(m => m.default && m.default.Chat)[0].default);
```

Comment:
Just an idea - can't we make check `window.Debug.VERSION` using `compareWwebVersions` and choose different stores?
like 
```js
   const pre2300 = window.compareWwebVersions(window.Debug.VERSION, '<', '2.2300.0')
   if (pre2300){
       # place old logic here
       window.Store = Object.assign({}, window.mR.findModule(m => m.default && m.default.Chat)[0].default);
       ... 
    } else {
        # place new logic here
        window.Store = Object.assign({}, window.mR.findModule(m => m && m.Chat)[1]);
        ...
    }
```

Or even 
```js
       const pre2300 = window.compareWwebVersions(window.Debug.VERSION, '<', '2.2300.0')
   if (pre2300){
       # place old logic here
       StoreInitializerNew(window)
       ... 
    } else {
        # place new logic here
        StoreInitializerOld(window)
        ...
    }
```

It'd look a bit better then a lot of logic in one line code... 

I'm happy to help with that if you need any!

Reply:
if using this [maunklana/moduleRaid/](https://github.com/maunklana/moduleRaid/)
you can check if WAWEB is Version 2.3000 or Older with this `moduleRaid.isComet` when is return `true` is WAWEB version 2.3000

yes you can

```
if (moduleRaid.isComet){
   # place newlogic here
   StoreInitializerNew(window)
   ... 
} else {
    # place old logic here
    StoreInitializerOld(window)
    ...
}
```

but I think is just more complicated to have more variable store or a separate function for each, I just following this project's typing style, or not?

Reply:
> just following this project's typing style

Yep, you are totally right in that, it's the move we should always make! 

>  is just more complicated 

Right now I see that that `moduleRaid.isComet` blurred implicitly in every changed lines and it's hard to understand "Which one is actually the right logic for the latest WWeb?" 

Let me see if I can show that it's actually easier approach and helpful for a long distance run with the code and not just words by creating another PR for that :)

Reply:
>Which one is actually the right logic for the latest WWeb?

To check the latest WAWEB you can use this 
`parseInt(window.Debug?.VERSION?.split(".")?.[1]) >= 3000` it will return `true` on Latest WAWEB

> Let me see if I can show that it's actually easier approach and helpful for a long distance run with the code and not just words by creating another PR for that :)

Sure... thanks.